<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>297</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>418</width>
    <height>297</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Menadżer haseł</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_master">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hasło główne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wyświetl listę haseł</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dodaj hasło</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove_password">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuń hasło</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>418</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_exit"/>
    <addaction name="separator"/>
    <addaction name="action_erase_everything"/>
    <addaction name="action_info"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>Zmień hasło główne</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Zamknij program</string>
   </property>
  </action>
  <action name="actionUsu_wszystkie_dane">
   <property name="text">
    <string>Usuń wszystkie dane</string>
   </property>
  </action>
  <action name="action_erase_everything">
   <property name="text">
    <string>Usuń wszystkie dane</string>
   </property>
  </action>
  <action name="action_info">
   <property name="text">
    <string>Informacje o programie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
